### DISCUSSION TITLE
Job description for analysis on blockchain

### DISCUSSION THREAD

**Discussion by kingfoxxx** [Cyfrin/foundry-full-course-cu#2982] [Q&A]
wanted learning a bit more about the Data analysis space on blockchain and web 3

**Comment by EngrPips** [comment]
Hello @kingfoxxx, I am not sure I understand your question.

**Reply by kingfoxxx** [reply]
Okayy sir
I’m actually a data analyst/data engineer trying to stretch into blockchain. Wanted knowing what space in blockchain aligns with data analysis and data in general.

**Reply by EngrPips** [reply]
I don't know much about data analysis to be able to correctly comprehend how it is applied in the web3 space, but I know that web3 is all about storing and getting data. If a field that heavily stores and retrieves data is something that interests you then web3 is a nice place for you.

**Reply by kingfoxxx** [reply]
Okayy 
Thankyou so much

**Comment by NikolayPIvanov** [comment]
Everything that provides or uses a historical data is in need of data engineer as the raw and transformed data can require quite the storage.
Here are some projects that might be interesting to you as an data engineer:
 - [ChainStorage](https://github.com/coinbase/chainstorage) - Coinbase's storage layer
 - [Reservoir](https://github.com/reservoirprotocol)  - NFT Marketplace / Aggregator, they provide historical and real-time NFT indexing with additional CDN and other features.
 - [Dune](https://dune.com/home) - great one for data analysis, I do not know if there are competitors

In terms of of opportunity -every company that wants to query and aggregate the blockchain data will be a fit and every serious company would like to do that. Minimizing cost is great, but preventing dependencies on third-party providers provides huge benefits in the long run.

**Reply by kingfoxxx** [reply]
Thankyou very, this is a path I can work on
